src/webgpu/shader/execution/robust_access_vertex.spec.ts

```diff
@@ -417,16 +509,111 @@ g.test('vertexAccess')
     pass.setPipeline(pipeline);
 
     // Run the draw variant
-    draw.insertInto(pass, p.indexed, p.indirect);
+    drawCall.insertInto(pass, isIndexed, isIndirect);
 
     pass.endPass();
-    t.device.queue.submit([encoder.finish()]);
+    this.device.queue.submit([encoder.finish()]);
 
-    // Validate we see green instead of red, meaning no fragment ended up on-screen
-    t.expectSinglePixelIn2DTexture(
+    // Validate we see green on the left pixel, showing that no failure case is detected
+    this.expectSinglePixelIn2DTexture(
       colorAttachment,
       'rgba8unorm',
       { x: 0, y: 0 },
       { exp: new Uint8Array([0x00, 0xff, 0x00, 0xff]), layout: { mipLevel: 0 } }
     );
+    // Validate wee see red on the right pixel, showing that at least one case succeed, prevent the test
+    // from unexpectly doing nothing and pass the validation
+    this.expectSinglePixelIn2DTexture(
+      colorAttachment,
+      'rgba8unorm',
+      { x: 1, y: 0 },
+      { exp: new Uint8Array([0xff, 0x00, 0x00, 0xff]), layout: { mipLevel: 0 } }
+    );
+  }
+}
+
+export const g = makeTestGroup(F);
+
+g.test('vertex_buffer_access')
+  .params(u =>
+    u
+      .combineWithParams([
+        { indexed: false, indirect: true },
+        { indexed: true, indirect: false },
+        { indexed: true, indirect: true },
+      ])
+      .expand('drawCallTestParameter', function* (p) {
+        if (p.indexed) {
+          yield* ['baseVertex', 'vertexCountInIndexBuffer'] as const;
+          if (p.indirect) {
+            yield* ['indexCount', 'instanceCount', 'firstIndex', 'firstInstance'] as const;
```

Comment:
FYI firstInstance should always be 0 in WebGPU for now for `drawIndirect` and `drawIndexedIndirect`. Can we remove them from these lists and add a TODO to add them back once there is an extension to allow making them non-0?

Reply:
Done.